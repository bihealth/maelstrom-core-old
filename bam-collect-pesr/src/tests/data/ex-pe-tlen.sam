../../../../bam-scan/src/tests/data/ex-pe-tlen.sam