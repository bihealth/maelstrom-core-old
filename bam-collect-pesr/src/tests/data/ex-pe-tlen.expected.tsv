#contig	start	end	signal
one	99	199	{"PairedRead":{"read_name":"pair_expected","is_first1":true,"contig1":"one","start1":99,"end1":199,"strand1":"Forward","contig2":"one","start2":499,"strand2":"Reverse","tlen":500}}
one	499	599	{"PairedRead":{"read_name":"pair_expected","is_first1":false,"contig1":"one","start1":499,"end1":599,"strand1":"Reverse","contig2":"one","start2":99,"strand2":"Forward","tlen":null}}
